### PR TITLE
Development

### PR DESCRIPTION
**Pull request by DerCalli**
Changelog:

- replace Spryker/MoneyPlugin by own converter
- adjust unit-tests
- AddToCartRenderer: Render JS to push added product to datalayer
- CartChangeQuantityRenderer: Render JS to push changed quantity of cart

### CODE REVIEW

File: src/FondOfSpryker/Yves/EnhancedEcommerceCartConnector/EnhancedEcommerceCartConnectorConfig.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+     * Returns list with array-keys which should not deleted even if they are empty
+     *
+     * @return array
+     */
+    public function getDontUnsetArrayIndex(): array
+    {
+        return $this->get(EnhancedEcommerceCartConnectorConstants::CONFIG_DONT_UNSET_ARRAY_INDEX, [
+            'action_field',
+        ]);
+    }
```
Comment (sourcecube) [inline src/FondOfSpryker/Yves/EnhancedEcommerceCartConnector/EnhancedEcommerceCartConnectorConfig.php]: A more suitable name could be `getArrayIndexWhitelist`

File: src/FondOfSpryker/Yves/EnhancedEcommerceCartConnector/EnhancedEcommerceCartConnectorDependencyProvider.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+     * @return \Spryker\Yves\Kernel\Container
+     */
+    protected function addStore(Container $container): Container
+    {
+        $container->set(static::STORE, static function (Container $container) {
+            return Store::getInstance();
+        });
+
+        return $container;
+    }
```
Comment (sourcecube) [inline src/FondOfSpryker/Yves/EnhancedEcommerceCartConnector/EnhancedEcommerceCartConnectorDependencyProvider.php]: Have you thought about using a client as well ?

**Review by sourcecube — APPROVED** [review]

File: src/FondOfSpryker/Yves/EnhancedEcommerceCartConnector/EnhancedEcommerceCartConnectorConfig.php (hunk 1 above)
Comment (DerCalli) [reply src/FondOfSpryker/Yves/EnhancedEcommerceCartConnector/EnhancedEcommerceCartConnectorConfig.php]: much better! 👌

**Review by sourcecube — APPROVED** [review]

**Review by julianhyatt — APPROVED** [review]

**Comment by julianhyatt** [comment]
tests/_data/twigVariableBag.php 30k lines... needed?

**Pull request merged**